rollup.config.js

```diff
@@ -232,3 +232,5 @@ export function makeMinificationVariants(existingConfigs) {
 
   return newConfigs;
 }
+
+export const rootBuildDirectory = 'build';
```

Comment:
```suggestion
export const ROOT_BUILD_DIR = 'build';
```

or

```suggestion
export const BUILD_DIR = 'build';
```

That way it's clear it's a constant.

Reply:
Right, makes sense. I think I'll go with option two. It's the shorter and simpler name.